### PR TITLE
Build dir fixed

### PR DESCRIPTION
**Pull request by eidheim**
Lets wait a day or two till we have gotten some feedback (positive or otherwise) on this solution

### CODE REVIEW

**Comment by eidheim** [comment]
And we'll also release v1.0.1 after merge. I'll write the changelog there.

File: src/cmake.cc [hunk 1]
```diff
@@ -33,19 +33,51 @@ CMake::CMake(const boost::filesystem::path &path) {
...
+    default_build_path=default_build_path_string;
+  
+  if(default_build_path.is_relative())
+    default_build_path=path/default_build_path;
+    
+  if(!boost::filesystem::exists(default_build_path)) {
+    boost::system::error_code ec;
+    boost::filesystem::create_directories(default_build_path, ec);
+    if(ec) {
+      Terminal::get().print("Could not create "+default_build_path.string()+": "+ec.message(), true);
```
Comment (eidheim) [inline src/cmake.cc]: Add "Error: " before "Could not ...."

**Comment by eidheim** [comment]
I have tried very hard to break this PR, and am out of ideas. It seems stable, no matter what I do. Merging it Tuesday morning and releasing version v1.0.1 then, if no issue arises that is.

**Comment by zalox** [comment]
- [ ] Ready for merge

**Comment by zalox** [comment]
I removed all the pull requests from this pull request.

**Comment by eidheim** [comment]
@zalox : You email account is down it seems, if you have another account could you send me a message to my gmail address?

**Pull request merged**